auth-api/src/auth_api/models/dataclass.py

```diff
@@ -47,6 +49,29 @@ class AffiliationInvitationSearch:  # pylint: disable=too-many-instance-attribut
     is_deleted: bool = False
 
 
+@dataclass
+class AffiliationInvitationSearchDetails:  # pylint: disable=too-many-instance-attributes
+    """Used for filtering Affiliation Invitations based on filters passed."""
+
+    identifier: Optional[str] = None
+    status: Optional[str] = None
+    name: Optional[str] = None
+    type: Optional[str] = None
+    page: int = 1
+    limit: int = 100
+
+    @classmethod
+    def from_request_args(cls, req: Request) -> Self:
+        return cls(
+            identifier=req.args.get("identifier"),
+            status=req.args.getlist("status") or [],
+            name=req.args.get("name"),
+            type=req.args.getlist("type") or [],
+            page=int(req.args.get("page", 1)),
+            limit=int(req.args.get("limit", 100)),
```

Comment:
Since the fallback values are provided here, can they be removed the the dataclass above?

Reply:
The initial values on lines 56-59 have been updated. Should lines 66-69 be updated similarly?

(I'm really wondering why there's 2 sets of initial/default values and why they don't match.)